governance/votes/Executive vote - November 14, 2024.md

```diff
@@ -19,6 +19,8 @@ If this executive proposal passes, the following **actions** will occur within t
 
 - The Stability Fee for multiple native vaults will be increased.
 - The Dai Savings Rate (DSR) and the Sky Savings Rate (SSR) will be increased.
+- 4 million USDS will be transferred for Sky Ecosystem liquidity bootstrapping funding.
+- 3 million USDS will be transferred for Integration Boost funding.
```

Comment:
```suggestion
- 3 million USDS will be transferred for integration boost funding.
```
Forum post does not use capital letters

Reply:
Capitalized in The Atlas - see https://github.com/makerdao/next-gen-atlas/blob/66a641557c01ad76a7cd0716698336d0fbc00505/Sky%20Atlas/Sky%20Atlas.html#L8768

Reply:
It is capitalised in The Atlas, so I'm going to pass this suggestion for now. We can add it to the list of items needing style guide decisions.